charts/addons/values.yaml

```diff
@@ -16,6 +16,11 @@ traefik:
     enabled: false
   rbac:
     enabled: true
+  # If Kubernetes namespace selection is defined and the (one) selected namespace is the release
```

Comment:
Are you essentially saying that addons should be deployed to a separate namespace and one addons installation could potentially work with several installations of Pega? In that case you would just omit (leave commented) the `kubernetes.namespaces` block from `values.yaml` since the `ClusterRole` and `ClusterRoleBinding` would be required in this case.

This change just gives the ability to have a 1-1 mapping of addons to pega in the event you want to completely isolate namespaces from one another. i.e not allowing a pre-prod installation of traefik to act as this Ingress for prod workloads.

Reply:
Yes. We recommend to have one installation of addons per cluster in a separate namespace, this would work for multiple installations of Pega. Having addons installed along with Pega in each namespace resulted in issues with traefik if nodes ports are not changed and with metrics server which allows only one installation per cluster.
If isolation of addons for each Pega deployment is the requirement you are looking for then we can have these changes and commented by default. Anyone utilizing this feature should take care of changing traefik node port in each namespace & enable metrics-server in only one namespace, disable in remaining.
Can you please update the comments so that it implies only one namespace can be given?

Reply:
Tag: @kpschuck